src/components/dagshub/data-engine/singleFileViewModal/TopButtonsSection.tsx

```diff
@@ -30,12 +37,28 @@ export default function TopButtonsSection({
           width: '100%',
           height: height,
           flexDirection: 'row',
+          gap: '5px',
           alignItems: 'center',
           justifyContent: 'space-between',
           flexShrink: 0,
           flexWrap: isSmallScreen ? 'wrap' : 'nowrap'
         }}
       >
+        {
+            (onSelectItem !== undefined && selected !== undefined) &&
+            <span>
+                <Tooltip
+                    content={"Disable “Select all” to choose specific items"}
+                    trigger={'click'}
+                    placement={'bottom'}
+                    interactive
```

Comment:
Doesn't really need to be interactive cause you don't need to click on the tooltip or copt text from it